### PR TITLE
fix(webkit): update Docker file to include gstreamer

### PR DESCRIPTION
**Pull request by yury-s**
https://github.com/microsoft/playwright/issues/2449

### CODE REVIEW

**Comment by arjun27** [comment]
Does the github action need an update? https://github.com/microsoft/playwright-github-action/blob/master/index.js

File: test/capabilities.spec.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ describe('Capabilities', function() {
     await page.goto(server.EMPTY_PAGE);
     expect(await page.evaluate(() => window.testStatus)).toBe('SUCCESS');
   });
-  it.fail(WEBKIT)('should play video', async({page, server}) => {
+  it.fail(WEBKIT && !LINUX)('should play video', async({page, server}) => {
```
Comment (aslushnikov) [inline test/capabilities.spec.js:50]: wait, we don't play video on mac / win?

**Review by aslushnikov — APPROVED** [review]

**Comment by aslushnikov** [comment]
> Does the github action need an update? https://github.com/microsoft/playwright-github-action/blob/master/index.js

Yep! we should theoretically see our tests failing until we update the github action as well

**Comment by yury-s** [comment]
> Does the github action need an update? https://github.com/microsoft/playwright-github-action/blob/master/index.js

I believe so, we'll probably need to bump its version too

File: test/capabilities.spec.js (hunk 1 above)
Comment (yury-s) [reply test/capabilities.spec.js:50]: According to Dmitry on mac there is a problem with our test server which doesn't support range query, so we should fix that. I will check if it works on windows and enable there separately.

Comment (yury-s) [reply test/capabilities.spec.js:50]: Reverting the test change for now, will enable it along with the roll.

**Pull request merged**